modules/ROOT/pages/update-acm.adoc

```diff
@@ -228,6 +229,32 @@ If you are upgrading from a package version before 1.79, update the permissions
 .. Ensure *Read* is enabled, and in *Field Permissions* ensure all fields have *Read Access* enabled.
 . Check that your permissions match those in the xref:create-community.adoc#configure-profiles-and-permissions[Configure Profiles and Permissions] documentation.
 
+Return to the main list of upgrade instructions to continue.
+
+[[post-1801]]
+== Post-Install Steps When Upgrading from a Version before 1.80.1
+
+If you are upgrading from a package version before 1.80.1, update the permissions of the `Member` and `Guest` profiles on the *AnypointGrantTypes* object, and ensure both `Member` and `Guest` profiles have field-level `Read` access to the `GrantTypes` field in the *AnypointApplications* object:
+
+. Configure the `Member` profile permissions:
+.. Click *Setup* > *Profiles*.
+.. Open the `Member` profile.
+.. Click *Object settings* > *AnypointGrantTypes*.
+.. Ensure *Read* is enabled, and in *Field Permissions* ensure all fields have *Read Access* enabled.
+.. Click *Object settings* > *AnypointApplications*.
+.. Ensure the `GrantTypes` field has `Read` access.
```

Comment:
Ensure that

See style ref. entry for "ensure"

Reply:
Fixed.